test/src/xquery/maps/mapsLookup.xql

```diff
@@ -0,0 +1,23 @@
+xquery version "1.0";
```

Comment:
@joewiz I notice it too. It's same way as other [test](https://github.com/eXist-db/exist/blob/develop/test/src/xquery/maps/maps.xql#L1)s too.